db/schema.rb

```diff
@@ -32,6 +32,39 @@
 
   add_index "activations", ["activation_token_id"], name: "index_activations_on_activation_token_id", unique: true, using: :btree
 
+  create_table "active_admin_comments", force: :cascade do |t|
```

Comment:
If the only thing that can be administered is users, and the only thing that can be made on them is disabling their accounts, is it really necessary to have an extra table just to store a comment about why it was disabled. It seems that this could be solved with a "disabled_reason" string variable in the user model.